test/opt/ir_loader_test.cpp

```diff
@@ -322,10 +321,9 @@ TEST(IrBuilder, KeepModuleProcessedInRightPlace) {
 // from the given |assembly|.
 void DoErrorMessageCheck(const std::string& assembly,
                          const std::string& error_message) {
-  auto consumer = [error_message](spv_message_level_t level, const char* source,
-                                  const spv_position_t& position,
-                                  const char* m) {
-    EXPECT_EQ(error_message, StringifyMessage(level, source, position, m));
+  auto consumer = [error_message](spv_message_level_t, const char*,
+                                  const spv_position_t&, const char* m) {
+    EXPECT_EQ(error_message, m);
```

Comment:
This change no longer checks that the error was found in the correct position.  It that really what we want?

Reply:
That could be why we ended up with such bad messages coming out of the validator.  I really think we should be checking the position as well.  I don't want these tests to check for less than they currently do.

Reply:
Done.